---
rail_semantic_grasping/SemanticObjectList semantic_objects